wiki/Contests/MGTS/2025/en.md

```diff
@@ -0,0 +1,112 @@
+---
+tags:
+  - mgts
+  - mgts2025
+  - mapping gts
+---
+
+# Mapping Global Taiko Showdown 2025
+
+![MGTS 2025 banner](img/banner.png)
+
+The **Mapping Global Taiko Showdown 2025** (***MGTS 2025***) is a single-round team-based osu!taiko mapping contest where teams of 2 to 3 members will be competing to map 2 out of 3 custom-made songs.
+
+## Contest schedule
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Registration and submission phase | 1 June – 29 June 23:59 |
+| Screening phase | *Tentative*[^screening] |
+| Judging phase | 30 June – 26 July |
+| Results | 27 July |
+
+## Organisation
+
+The MGTS 2025 is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Hosts | ::{ flag=TN }:: [**Hivie**](https://osu.ppy.sh/users/14102976), ::{ flag=SG }:: [arcpotato](https://osu.ppy.sh/users/12842392), ::{ flag=MY }:: [Jerry](https://osu.ppy.sh/users/605973) |
+| Designer | ::{ flag=MY }:: [Z419](https://osu.ppy.sh/users/9912966) |
+| Judges | ::{ flag=DE }:: [Greenshell](https://osu.ppy.sh/users/8693851), ::{ flag=HK }:: [iRedi](https://osu.ppy.sh/users/8005579), ::{ flag=MY }:: [ler1211](https://osu.ppy.sh/users/19901680), ::{ flag=US }:: [meiqth](https://osu.ppy.sh/users/12565402), ::{ flag=AU }:: [Yasuho](https://osu.ppy.sh/users/8458835) |
+
+## Prizes
+
+The top 3 teams will be awarded with the following prizes:
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge (pending), [contest points](/wiki/Contests/Contest_points) (pending), Ranked status[^prizes-ranked] |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | [contest points](/wiki/Contests/Contest_points) (pending), Ranked status[^prizes-ranked] |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | [contest points](/wiki/Contests/Contest_points) (pending), Ranked status[^prizes-ranked] |
+
+## Links
+
+- [Main website](https://gtsosu.com/2025/mgts/home)
+- [Contest news article](https://osu.ppy.sh/news/2025-06-01-mgts-2025-registrations)
+- [Contest listing](https://osu.ppy.sh/community/contests/248)
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/2086331)
+- [Discord server](https://discord.com/invite/3mGC3HB)
+- [Twitch channel](https://www.twitch.tv/gtsosu)
+
+## Songs
+
+| Song | Artists |
```

Comment:
bpm and length?

Reply:
I plan to rework this section later when the songs are fully released on their designated platforms, so I'll keep this in mind for when i do that.